Lib/test/test_capi/test_misc.py

```diff
@@ -298,6 +298,86 @@ def test_getitem_with_error(self):
             # test _Py_CheckFunctionResult() instead.
             self.assertIn('returned a result with an exception set', err)
 
+    def test_buildvalue(self):
+        """Test Py_BuildValue() with object arguments"""
```

Comment:
These docstrings pollute the test output when run in verbose mode. Please don't add these. Regular comments do not pollute the test output.